src/utils/aepp-wallet-communication/schema.ts

```diff
@@ -20,6 +20,7 @@ export const enum METHODS {
   updateAddress = 'address.update',
   address = 'address.get',
   connect = 'connection.open',
+  bridge = 'connection.bridge',
```

Comment:
"bridge" sounds too confusing, can you find a better name?

Reply:
We have two techniques.
1. Bridge/ConnectNode/linkNode/attachNode/addNode: I guess attachNode makes more sense
2. Proxy/delegate

@davidyuk